tests/reports.test.js

```diff
@@ -0,0 +1,345 @@
+            .auth(accessToken, { type: 'bearer' })
+            .end( (error, response) => {
+                response.should.have.status(200);
+                response.body.should.be.a("array");
+                response.body.length.should.be.eq(2);
+                done();
+            });
+        });
+    
+        it("GET /Reports NameAccount Not Found", (done) =>{
+            chai.request(server).get("/reports/nameAccount/Mariana")
+            .auth(accessToken, { type: 'bearer' })
+            .end( (error, response) => {
+                response.should.have.status(404);
+                response.body.should.have.property("message");
+                response.body.should.have.property("message").eq("No se encontro registro(s)");
+                done();
+            });
+        });
+    
+        it("GET /Reports NameAccount Bad Request", (done) =>{
+            chai.request(server).get("/reports/nameAccount/12352")
+            .auth(accessToken, { type: 'bearer' })
+            .end( (error, response) => {
+                response.should.have.status(400);
+                response.body.should.have.property("message");
+                response.body.should.have.property("message").eq("Solo letras de la A a la Z. Caracteres de 2 a 150");
+                done();
+            });
+        });
+    });
+
+    describe("GET Tests Reports NameReported", () => {
+        it("GET /Reports NameReported", (done) =>{
+            chai.request(server).get("/reports/nameReported/Mariana")
+            .auth(accessToken, { type: 'bearer' })
+            .end( (error, response) => {
+                response.should.have.status(200);
+                response.body.should.be.a("array");
+                response.body.length.should.be.eq(2);
+                done();
+            });
+        });
+    
+        it("GET /Reports NameReported Not Found", (done) =>{
+            chai.request(server).get("/reports/nameReported/Miroslava")
+            .auth(accessToken, { type: 'bearer' })
+            .end( (error, response) => {
+                response.should.have.status(404);
+                response.body.should.have.property("message");
+                response.body.should.have.property("message").eq("No se encontro registro(s)");
+                done();
+            });
+        });
+    
+        it("GET /Reports NameReported Bad Request", (done) =>{
+            chai.request(server).get("/reports/nameReported/12352")
+            .auth(accessToken, { type: 'bearer' })
+            .end( (error, response) => {
+                response.should.have.status(400);
+                response.body.should.have.property("message");
+                response.body.should.have.property("message").eq("Solo letras de la A a la Z. Caracteres de 2 a 150");
+                done();
+            });
+        });
+    });
+
+    describe("GET Tests Reports LastnameReported", () => {
+        it("GET /Reports LastnameReported", (done) =>{
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
        it("GET /Reports LastnameReported", (done) => {
```